### ISSUE TITLE
Add support for Eth+Zil+Alph

### ISSUE THREAD

**Issue by GrzegorzWo** [trexminer/T-Rex#1067]
Dear Dev Team,
Now, when we have working support for Eth+Alph mining, the next, most urgent feature should be to combine Eth+Zil with Alph. It seems it does not require much effort (of course adding --url3 and --user3 params for blake algorithm will be necessary, but rest of functionality should be simple merge of Eth+Alph into Eth+Zil).

Eth+Zil+Alph feature would give free choose for any pool for any coin. Also this will ensure best performance for mining of all three coins.

At the moment the only possibility to triple mine is to:

1. use ezli.me pool for both Eth and Zil, 
2. use Ezil pool, which supports Eth Proxing.

Both solutions don't work well at the moment and free selection of all three pools is obligatory next feature.

**Comment by mflanagan1041** [comment]
I agree with this. It seems like the problems lie with the miner switching from eth to zil and alph at the same time. please look into adding this. some of my cards with windows id prefer to mine 3 coins over 2 especially when the reduced megahash is negligent. 

also make sure it works for non lhr cards to.

**Comment by GrzegorzWo** [comment]
If mentioned on my ticket, as an update I can add:

Tripple mining (Eth+Zil+Aplh) is possible now via proxy pool, as written above. However, I was not able t-rex to use "extra-dag-epoch": "0" parameter. This means when tripple mining, DAG is being generated each time Zilliqa session is started and finished.
It causes unnecessary delays in minig.

I guess it is because during Alph mining the "--lhr-algo" parameter is used. This makes, we can't use "coin": "eth+zil" any more. This makes "extra-dag-epoch": "0" is being ignored.

So the good starting point to improve current Eth+Zil+Alph mining is to make "extra-dag-epoch": "0" not being ignored.
